exercises/say/SayTest.cs

```diff
@@ -8,90 +8,90 @@ public class SayTest
     [Fact]
     public void Zero()
     {
-        Assert.Equal("zero", Say.InEnglish(0));
```

Comment:
I've now made it clear that each input value is actually a long value. Previously, an implicit conversion was taking place, but this is more clear I feel.